styles/blocks/f-button-group.css

```diff
@@ -3,33 +3,11 @@
 A container that put 3 buttons per row with equal space between them, where the first and last button are justified to the edge.
 Goes into two columns on mobile. 
 
+<pre>
 `BLOCK f-button-group`
     `ELEMENT f-button-group__button`
     `MODIFIER f-button-group__button_selected`
-
-## Button effects
-
-So yeah, it's hard to choose. The simple one? The minimalistic underline? The big sweeping one? 
-
-```
-<div class="kalei-background-white">
-<!-- only here for the style guide - should not be used on the website -->
-<!-- this should be used on the website -->
-
-	<section class="f-button-group">
-		<div class="f-column f-column_3 f-column_mobile_2">
-			<button class="f-button-group__button_fade">Terms</button>
-			<button class="f-button-group__button_underline">Blog</button>
```

Comment:
Karl BEM recap:
```
BLOCK f-button-group
ELEMENT f-button-group__button
MODIFIER f-button-group__button_stop
```
A **Block** can stand alone. An **Element** must be in a **Block**. A **Modifier** can only be on a **Block** or **Element** and can **NEVER** stand alone. So `<button class="f-button-group__button_stop">Manifesto</button>` is **not** possible. It's a **Modifier** within a **Block** but not on an **Element**.
The correct version would be along these lines:
```html
<section class="f-button-group">
	<div class="f-column f-column_3 f-column_mobile_2">
		<button class="f-button-group__button f-button-group__button_fade">Terms</button>
```